templates/footer.html

```diff
@@ -0,0 +1,5 @@
+				</div>
```

Comment:
I think having HTML elements open in one file and close in another, while perfectly possible from a technical point of view, creates a lot of issues later. One example is Wordpress themes. It is common to open and close tags in different files there, ultimately leading to confusion. 

Are you sure this is required for a small file like `index.html`? I know it will grow up over time, but how reusable will this code (`footer.html`) be?